compiler/rustc_ast/src/ast.rs

```diff
@@ -167,6 +167,8 @@ pub enum GenericArgs {
     AngleBracketed(AngleBracketedArgs),
     /// The `(A, B)` and `C` in `Foo(A, B) -> C`.
     Parenthesized(ParenthesizedArgs),
+    /// Return-type notation: `(..)`
```

Comment:
request: could this have a more detailed comment, like the other ones do, showing it in context?  Just repeating the name of the variant doesn't help me much.